File: app/backend/models/event.js
const {mongoose} =  require('../db');  // The mongodb connector library

let Event = mongoose.model('Event', {
  calendarId: {
    type: String
  },

  event: {
    type: String
  },

  date: {
    type: Date
  },

  source: {
    type: String
  },

  country: {
    type: String
  },

  importance: {
    type: Number
  },

  actual: {
    type: Number
  },

  previous: {
    type: Number
  },

  forecast: {
    type: Number
  },
});

module.exports = {
  Event: Event
}

File: app/android/app/src/main/java/com/example/arken/model/Event.kt
package com.example.arken.model

import java.io.Serializable
import java.util.*

class Event(
    val _id: String?,
    val CalendarId: String?,
    val Country: String?,
    val Date: Date?,
    val Event: String?,
    val Source: String?,
    val Actual: String?,
    val Previous: String?,
    val Forecast: String?,
    val Importance: Int?,
    val __v: String?,
    val comments: MutableList<Comment>
) : Serializable